client/lib/signup/flow-controller.ts

```diff
@@ -307,9 +307,10 @@ export default class SignupFlowController {
 			We are testing whether a passwordless account creation and login improves signup rate in the `onboarding` flow
 		*/
 		if (
-			'onboarding' === this._flowName &&
-			get( step, 'isPasswordlessSignupForm' ) &&
-			'passwordless' === abtest( 'passwordlessSignup' )
+			( 'test-fse' === this._flowName && get( step, 'isPasswordlessSignupForm' ) ) ||
```

Comment:
Could we remove the flow and AB checks here (and just use `get( step, 'isPasswordlessSignupForm' )`) and assume that if a user got the passwordless signup form, it was meant to be? I might be wrong, but let's say that somehow a user got the form, and wasn't in the test or flow, then Signup would break at the user step.

Reply:
Great idea, thanks @michaeldcain I've updated this to a simple check for just that step property — feels like we're less likely to accidentally break things this way :)